### PR TITLE
RHDEVDOCS-6172: Minor changes in the Argo CD CR properties table

### PR DESCRIPTION
**Pull request by Dhruv-Soni11**
Version(s):

GitOps 1.13, GitOps 1.14

Issue:

https://issues.redhat.com/browse/RHDEVDOCS-6172

Link to docs preview:

Changed the properties in both tables alphabetically.
Added the `sourceNamespaces` property in the Argo CD custom resource properties table
Removed the `<` and `>` signs from all the user-defined values in both the tables.

[Argo CD custom resource properties](https://82753--ocpdocs-pr.netlify.app/openshift-gitops/latest/argocd_instance/argo-cd-cr-component-properties.html)

QE review:
- [x] QE has approved this change.

SME review: @svghadi 
QE review: @varshab1210 
Peer review: @mburke5678 

Additional information:

### CODE REVIEW

**Review by svghadi — APPROVED** [review]

**Comment by varshab1210** [comment]
LGTM

File: modules/gitops-argo-cd-properties.adoc [hunk 1]
```diff
@@ -16,160 +16,162 @@ The Argo CD Custom Resource consists of the following properties:
...
+  * `sharding.replicas` - The number of replicas that are used to support sharding of the Argo CD Application Controller.
+  * `sharding.dynamicScalingEnabled` - Enables the dynamic scaling of the Argo CD Application Controller component. Use this property if you want the Operator to scale the number of replicas based on the number of clusters the controller component is managing. If you set this property to `true`, it overrides the configuration of the `sharding.enabled` and `sharding.replicas` properties.
+  * `sharding.minShards` - The minimum number of Argo CD Application Controller replicas.
+  * `sharding.maxShards` - The maximum number of Argo CD Application Controller replicas.
+  * `sharding.clustersPerShard` - The number of clusters that need to be managed by each shard. When the replica count reaches the `maxShards`, the shards manage more than one cluster.
+  * `env` - Environment to set for the application controller workloads.
+  * `sourceNamespaces` - The list of non-control plane namespaces for creating and managing Argo CD `Application` resources in target namespaces.
```
Comment (varshab1210) [inline modules/gitops-argo-cd-properties.adoc:52]: @Dhruv-Soni11 please remove this field from .spec.controller

cc: @svghadi

Comment (Dhruv-Soni11) [reply modules/gitops-argo-cd-properties.adoc:52]: Fixed.

Comment (Dhruv-Soni11) [reply modules/gitops-argo-cd-properties.adoc]: Fixed.

**Comment by Dhruv-Soni11** [comment]
> @Dhruv-Soni11 LGTM. The alphabetical order is based only on the first letter, rather than all letters. probably not worth fussing with.

Hey @mburke5678, I am paying heed to this request and I've rearranged all the properties and sub-properties alphabetically as I have little bit of OCD in me too.

**Comment by Dhruv-Soni11** [comment]
Hey @xenolinux, as per our discussion on Slack we do not need to merge the changes into the GitOps 1.12 branch as the SME has suggested that version will not be supported. Also, I have included both the PRs for the cherry-picks in this PR. Please let me know if you need any other information.

**Pull request merged**